utilities/transactions/write_unprepared_transaction_test.cc

```diff
@@ -182,8 +182,8 @@ TEST_P(WriteUnpreparedStressTest, ReadYourOwnWriteStress) {
     ReadOptions read_options;
 
     for (uint32_t i = 0; i < kNumIter; i++) {
-      std::set<std::string> owned_keys(&keys[id * kNumKeys],
-                                       &keys[(id + 1) * kNumKeys]);
+      std::set<std::string> owned_keys(keys.begin() + id * kNumKeys,
+                                       keys.begin() + (id + 1) * kNumKeys);
```

Comment:
this iterator can still go out of bounds, can we explicitly compute a valid end?

Reply:
I think it's fine for iterator as keys.begin() + keys.size() = keys.end().

Reply:
correct, after reading the context, this cannot go over keys.end, so it's fine.